### PR TITLE
Add script to generate BOM File

### PR DESCRIPTION
**Pull request by mathuriga**
Checkout version
   Create BOM file
   Let user to upload BOM file
   Delete cloned repository.

## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: internal/dependency-track/GenerateBOMFile.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+################################################################
+##   This script will generate a BOM file for given product   ##
+################################################################
+
+# ------------------------------------
+# Download source code
+# ------------------------------------
+
+read -p "Enter git source code download url to create BOM File : " download_url
```
Comment (ayomawdb) [inline internal/dependency-track/GenerateBOMFile.sh]:
```suggestion
read -p "Enter URL to download source code: " download_url
```

File: internal/dependency-track/GenerateBOMFile.sh [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+mkdir $HOME/workingDir
```
Comment (ayomawdb) [inline internal/dependency-track/GenerateBOMFile.sh]: shall we do this in `/tmp` and then delete the folder at the end? `/tmp/bom-generator`

File: internal/dependency-track/GenerateBOMFile.sh [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+cd $HOME/workingDir
+wget $download_url
+
+# --------------------------------------
+# Unzip source
+# --------------------------------------
+
+zip_file_name=$(ls )
+mkdir sourceDir
+unzip $HOME/workingDir/$zip_file_name -d $HOME/workingDir/sourceDir
```
Comment (ayomawdb) [inline internal/dependency-track/GenerateBOMFile.sh]: `/tmp/bom-generator/src`?

File: internal/dependency-track/GenerateBOMFile.sh [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+nautilus $HOME/workingDir/sourceDir/$file_name/target
+
+sleep 60
+read -n 1 -r -s -p "If you have uploaded BOM file to Dependency Track successfully, Press any key to continue..."
+
+# ------------------------
+# Delete Cloned Repository
+# ------------------------
+
+rm -rf $HOME/workingDir/
```
Comment (ayomawdb) [inline internal/dependency-track/GenerateBOMFile.sh]: rm the `/tmp/bom-generator/src` folder leaving the BOM file in `/tmp/bom-generator/example.bom`.. Later on we can provide an option `-o` to define the output folder.

File: internal/dependency-track/GenerateBOMFile.sh (hunk 1 above)
Comment (mathuriga) [reply internal/dependency-track/GenerateBOMFile.sh]: Done.

File: internal/dependency-track/GenerateBOMFile.sh (hunk 2 above)
Comment (mathuriga) [reply internal/dependency-track/GenerateBOMFile.sh]: Done

File: internal/dependency-track/GenerateBOMFile.sh (hunk 3 above)
Comment (mathuriga) [reply internal/dependency-track/GenerateBOMFile.sh]: Done

File: internal/dependency-track/GenerateBOMFile.sh (hunk 4 above)
Comment (mathuriga) [reply internal/dependency-track/GenerateBOMFile.sh]: Done,

**Pull request merged**